modules/telco-ran-lvms-operator.adoc

```diff
@@ -1,33 +1,25 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+// * scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="telco-ran-lvms-operator_{context}"]
-= {lvms}
+= Logical Volume Manager Storage
 
 New in this release::
 * No reference design updates in this release
 
-[NOTE]
-====
-{lvms-first} is an optional component.
-
-When you use {lvms} as the storage solution, it replaces the Local Storage Operator.
-CPU resources are assigned to the management partition as platform overhead.
-The reference configuration must include one of these storage solutions, but not both.
-====
-
 Description::
-{lvms} provides dynamic provisioning of block and file storage.
-{lvms} creates logical volumes from local devices that can be used as `PVC` resources by applications.
+{lvms-first} is an optional component.
+It provides dynamic provisioning of both block and file storage by creating logical volumes from local devices that can be consumed as `PVC` resources by applications.
 Volume expansion and snapshots are also possible.
+An example configuration is provided in the RDS with the `StorageLVMCluster.yaml` CR.
```

Comment:
I am probably overthinking it, but when I see an actual file name of the example with a file extension like this, it makes me think to call it a "file":
```suggestion
An example configuration is provided in the RDS with the `StorageLVMCluster.yaml` file.
```
Even though it is true that within that file is an example CR. Up to you but thought I'd mention.